jekyll/_cci2/runner-overview.adoc

```diff
@@ -9,7 +9,7 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: The CircleCI runner is currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan or to learn more.
+NOTE: The CircleCI runner is available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan or to learn more.
```

Comment:
1. your new wording from a similar statement in the other file sounds cleaner: "CircleCI runner is currently available on the..."
2. Does it make sense to push interested readers to reach out if they don't have a sales rep:  "Please reach to your sales representative or contact us <linked to https://circleci.com/contact-us/?cloud> for information on ..."?